pootle/static/js/shared/components/lightbox.css

```diff
@@ -101,18 +115,19 @@ html[dir="rtl"] .lightbox-footer .btn + .btn
     position: absolute;
     right: 0;
     top: 0;
-    width: 44px;
-    height: 44px;
-    line-height: 44px;
+    width: 30px;
+    height: 100%;
+    line-height: 100%;
+    padding: 0;
+    margin: 0;
 
     text-decoration: none;
     text-align: center;
     background-color: transparent;
     color: rgba(0, 0, 0, 0.5);
 
     font-style: normal;
-    font-size: 28px;
-    font-family: "Arial", monospace;
+    f--ont-size: 14px;
```

Comment:
that's how I quickly disable some CSS rules during development; forgot to remove before committing. It should not affect rendering, I'll remove it.